docs/api.md

```diff
@@ -33,6 +33,7 @@ Functions for running experiments.
     experiments.load_and_run_regression_experiment
     experiments.run_clustering_experiment
     experiments.load_and_run_clustering_experiment
+    experiments.forecasting_experiments.run_experiment
```

Comment:
Recursively search through `tsml_eval/` for public files to add to the API page. If possible, use the import from the package `__init__.py` file.